pepper-apis/config/testing-inmemorydb.conf.ctmpl

```diff
@@ -107,6 +107,7 @@
         "signerServiceAccount": {{$fileUploads.signerServiceAccount | toJSONPretty}}
     },
     "fileDownload":{
+        "credentials": {{$conf.Data.fileDownload.signerServiceAccount | toJSONPretty}},
```

Comment:
Heads up @pegahtah - Between PEPPER-890 and this PR, there are some changes to the fileDownload "testing" configuration.  My addition at 110 ensures the credentials object contains what DSM expects.  PEPPER-890 added what DSS expects.
When I took develop's changes after 890 merged, I got tripped up by the re-rendering.